docs/UserGuide.md

```diff
@@ -86,15 +131,65 @@ This section contains the documentation on NurseyBook's features and commands. I
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
```

Comment:
Perhaps we can change these parameters to ones that NurseyBook uses? (re: lines 123 and 132)